doc/man/man3/seccomp_export_bpf.3

```diff
@@ -13,6 +13,7 @@ seccomp_export_bpf, seccomp_export_pfc \- Export the seccomp filter
 .sp
 .BI "int seccomp_export_bpf(const scmp_filter_ctx " ctx ", int " fd ");"
 .BI "int seccomp_export_pfc(const scmp_filter_ctx " ctx ", int " fd ");"
+.BI "int seccomp_export_bpf_mem(const scmp_filter_ctx " ctx ", void *" buf ", size_t *" len ");"
```

Comment:
In addition to the documenting the new function prototype here, the **DESCRIPTION** section also needs some text describing the new `seccomp_export_bpf_mem()` API.  Beyond the usual, I would expect the description to explain that callers should pass the length of `buf` in `len` when calling the function, but can determine the useful length of `buf` by inspecting `len` when the function returns `0` or `-ERANGE\-E2BIG` (see comments below).